### PR TITLE
Benji/ios 242 threaded presentation

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/Flows/Onboarding/OnboardingViewController.swift [hunk 1]
```diff
@@ -23,10 +23,6 @@ class OnboardingViewController: SwitchableContentViewController<OnboardingConten
         return .fade
     }
 
-    var transitionColor: Color {
-        return .background
```
Comment (Benjamindodgson) [inline Shared/Flows/Onboarding/OnboardingViewController.swift:27]: removing this.

File: Shared/Models/Message Model/MessageSequence.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+//
+//  Created by Benji Dodgson on 11/17/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+protocol MessageSequence {
+
+    var id: String { get }
```
Comment (Benjamindodgson) [inline Shared/Models/Message Model/MessageSequence.swift:13]: new protocol

File: iOS/Flows/Archives/ArchiveCoordinator.swift [hunk 3]
```diff
@@ -133,10 +133,13 @@ extension ArchiveCoordinator: ArchiveViewControllerDelegate {
             self.addChildAndStart(coordinator, finishedHandler: { [unowned self] (_) in
                 self.router.dismiss(source: self.archiveVC, animated: true)
             })
-            self.router.present(coordinator,
-                                source: self.archiveVC,
-                                cancelHandler: {
-            })
+            self.router.push(coordinator, cancelHandler: {
```
Comment (Benjamindodgson) [inline iOS/Flows/Archives/ArchiveCoordinator.swift:136]: Ill be working on this in a future PR

File: iOS/Flows/Conversation/Cells/MessageContentView.swift [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+//  MessageContentView.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 11/17/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class MessageContentView: View {
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Cells/MessageContentView.swift:11]: I moved this to its own view so I could reuse it

File: Shared/UI/Transitions/TransitionRouter+Move.swift [hunk 5]
```diff
@@ -19,6 +20,8 @@ extension TransitionRouter {
             return
         }
 
+        let foo = transitionContext.viewController(forKey: .from)
```
Comment (MartinJibber) [inline Shared/UI/Transitions/TransitionRouter+Move.swift]: Looks like leftover test code.

**Review by MartinJibber — CHANGES_REQUESTED** [review]
Looks great, just some testing code left in. Remove that and you're good to merge.

**Pull request merged**